internal/hcsoci/hcsdoc_wcow.go

```diff
@@ -397,3 +441,20 @@ func parseAssignedDevices(ctx context.Context, coi *createOptionsInternal, v2 *h
 	v2.AssignedDevices = v2AssignedDevices
 	return nil
 }
+
+// parseDumpType parses the passed in string representation of the local user mode process dump type to the
+// corresponding value the registry expects to be set.
+func parseDumpType(annotations map[string]string) (int32, error) {
+	dmpTypeStr := annotations[oci.AnnotationWCOWProcessDumpType]
+	switch dmpTypeStr {
+	case "":
+		// If no type specified, default to full dumps.
+		return 2, nil
```

Comment:
are these values documented? should we add a link?

Reply:
Yep, I can add a link here